app/templates/events/view/tickets/access.hbs

```diff
@@ -0,0 +1,74 @@
+{{#if device.isMobile}}
```

Comment:
I have pushed to heroku. The link is same as before. Please check UI and tell me if I should make the changes you told